tests/sqlparser_mssql.rs

```diff
@@ -570,6 +570,41 @@ fn parse_substring_in_select() {
     }
 }
 
+#[test]
+fn parse_mssql_execute_stored_procedure() {
```

Comment:
It is mssql specific. But I'd prefer not gating the behavior. In general, when a particular dialect's syntax is not incompatible with other dialects, I think it's better to always accept parsing it. Library users are always free to reject successfully parsed ASTs later, but when something fails to parse, you have fewer options.

Reply:
Not gating the behavior sounds reasonable! I think we should move the tests to common in that case so that the tests cover the parser behavior. otherwise could happen that some variant is being correctly parsed for some dialect being relied on but since we have no test coverage for that dialect, we could end up breaking it in the future if guards are added or the code changes. Also currently its a bit hidden like with the `has_parenthesis` thread not knowing which dialects are valid for which variants of the statement, even though the parser accepts both.

Reply:
I moved the tests to common :)